starboard/nplb/compiler_compliance/cpp17_support.cc

```diff
@@ -14,9 +14,6 @@
 
 #include <utility>
 
-#include "starboard/configuration.h"
-#include "starboard/types.h"
```

Comment:
I think at some point starboard/types.h might've been used to set int/char size but that might've since been removed by @yell0wd0g ?

Reply:
Yeah I figured `starboard/configuration.h` had likely just been copied over from some other related file that used `SB_COMPILE_ASSERT` instead of `static_assert`, like `starboard/nplb/compiler_compliance/cpp14_constexpr.cc`. But I was less sure about `starboard/types.h`. I don't really see why we'd want to include it there but I may be missing something.

Reply:
It looks good to me if it compiles, but have Miguel take a look to make sure we're not missing anything